libs/constrained-generators/constrained-generators.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               constrained-generators
-version:            0.2.0.0
+version:            0.2.1.0
```

`constrained-generators` are not being released anywhere yet, so we do not need to change its version yet.

Also, a change to bounds would have only required a patch version bump.
```suggestion
version:            0.2.0.0
```